tasktiger/task.py

```diff
@@ -228,7 +229,7 @@ def _move(self, from_state=None, to_state=None, when=None, mode=None):
         try:
             pipeline.execute()
         except redis.ResponseError as e:
-            if '<FAIL_IF_NOT_IN_ZSET>' in unicode(e):
+            if '<FAIL_IF_NOT_IN_ZSET>' in text_type(e):
```

Comment:
`if '<FAIL_IF_NOT_IN_ZSET>' in e.args[0]:` should work, too, and wouldn't require `six`.